test/verify/check-sosreport

```diff
@@ -38,15 +38,21 @@ class TestSOS(MachineCase):
         m.write("/etc/sos.conf",
                 """
 [general]
-only-plugins=release
+only-plugins=release,date,host
+threads=2
 """)
 
         self.login_and_go("/sosreport")
 
         b.click('[data-target="#sos"]')
         b.wait_visible("#sos")
 
+        def get_width():
+            return float(b.attr(".progress-bar", "style").split(":")[1][:-2])
```

Comment:
Oh, this looks super magic.  I was thinking of something like
 ```
b.eval_js('ph_find(".progress-bar").offsetWidth')
```

Reply:
Heh, yours is much much nicer. Updated to that.